core/commands/ssh_command.rb

```diff
@@ -0,0 +1,96 @@
+# frozen_string_literal: true
+
+require_relative 'base_command'
+require_relative '../models/configuration'
+require_relative '../models/result'
+require_relative '../services/terraform_service'
+require_relative '../services/vagrant_service'
+
+# The command executes a command on the virtual machine via ssh.
+class SshCommand < BaseCommand
+  def self.synopsis
+    'Execute a command on the virtual machine via ssh.'
+  end
+
+  def show_help
+    info = <<-HELP
+'ssh' executes a command on the virtual machine via ssh.
+
+mdbci ssh --command "ls ~" T/node0 - execute a command on the T/node0
```

Comment:
Put the comments to the invocation command before the command. Currently it seems that the command is just too long.